sherlocked.js

```diff
@@ -0,0 +1,74 @@
+var sherlocked = require('sherlocked');
+
+var PORT = require('./config').PORT;
+
+var environments = [
+    {
+        browserName: 'firefox',
+        platform: 'OS X 10.9'
+    },
+    {
+        browserName: 'chrome',
+        platform: 'OS X 10.9'
+    }
+];
+
+
+var captures = [
+    {
+        name: 'Home Page on Mobile',
+        capture: function(client) {
+            return client.setViewportSize({width: 320, height: 960})
+                .url('http://localhost:' + PORT)
+                .waitForExist('.feed-home', 180000);
```

Comment:
Is that 3 minutes? Will this ever take 3 minutes to load? One minute seems like lots of time.